scammer-label-propagation/src/agent.py

```diff
@@ -0,0 +1,171 @@
+import logging
+import os
+import time
+import forta_agent
+from web3 import Web3
+from forta_agent import get_json_rpc_url, Finding
+from concurrent.futures import ThreadPoolExecutor
+
+from src.main import run_all
+from src.constants import attacker_bots, ATTACKER_CONFIDENCE, N_WORKERS, MAX_FINDINGS
+
+
+# If we are in production, we log to the console. Otherwise, we log to a file
+if 'NODE_ENV' in os.environ and 'production' in os.environ.get('NODE_ENV'):
+    logging.basicConfig(level=logging.INFO, 
+                        format='%(levelname)s:%(asctime)s:%(name)s:%(lineno)d:%(message)s')
+else:
+    logging.basicConfig(filename=f"logs.log", level=logging.INFO, 
+                        format='%(levelname)s:%(asctime)s:%(name)s:%(lineno)d:%(message)s')
+logger = logging.getLogger(__name__)
+
+web3 = Web3(Web3.HTTPProvider(get_json_rpc_url()))
+
+
+def run_all_extended(central_node, alert_event):
+    try:
+        attackers_df, graph_statistics = run_all(central_node)
+    except Warning as w:
+        logger.warning(f"{central_node}:\tWarning running run_all in a thread: {w}", exc_info=True)
+        return []
+    except Exception as e:
+        logger.error(f"{central_node}:\tError running run_all in a thread: {e}", exc_info=True)
+        # We need to raise the exception to expose error to the scan node
+        if 'NODE_ENV' in os.environ and 'production' in os.environ.get('NODE_ENV'):
```

Comment:
you can just do this
```suggestion
if 'production' in os.environ.get('NODE_ENV', ''):
```

Reply:
corrected